lang/swi-prolog-devel/Portfile

```diff
@@ -7,7 +7,7 @@ PortGroup           compiler_blacklist_versions 1.0
 name                swi-prolog-devel
 conflicts           swi-prolog swi-prolog-lite
```

Comment:
the `swi-prolog-lite` port doesn't exist anymore - removed in 5b78527791d23667665d3b639be4fe9d9703bb88. Can you please remove that port from the `conflicts` line here?